functions/lambda-version-cleaner/app.py

```diff
@@ -97,7 +97,7 @@ def lambda_handler(event, context):
 
         # Loop through all functions and remove old versions
         for function in functions:
-            versions = get_function_versions(function, region)
+            versions = sorted(get_function_versions(function, region), key=int, reverse=True)
```

Comment:
You are already returning the list sorted in `get_function_versions` so make this fix in there....

Reply:
I'd also like to see the test data in this PR so we know it will actually work :)